Solutions/AzureKubernets/Analytic Rules/AKS-Rbac.yaml

```diff
@@ -0,0 +1,25 @@
+id: 765dce51-689a-44b5-9b38-c9ae9441c7fc
+name: Azure RBAC AKS created role details
+description: |
+  'Query get the details of role created for kube-audit.
```

Comment:
It is not clear why this would be a detection; can you provide more details as to why this would be?  If so, please update the description to make this clearer as to how this is something specifically malicious.

Reply:
@shainw  - Changed to hunting query